app/views/layouts/application.html.erb

```diff
@@ -10,9 +10,9 @@
 
   <body>
     <nav>
-      <div class="nav-wrapper">
+      <div class="nav-wrapper black">
         <a href="#" class="brand-logo">Back of the Yards Coffee Co.</a>
-        <ul id="nav-mobile" class="right hide-on-med-and-down">
+        <ul id="nav-mobile black" class="right hide-on-med-and-down">
```

Comment:
I'm totally fine with this change, but it appears outside scope of the current task of adding the RSPEC test.  It's obviously not a big deal in this case, but it's good practice to only submit changes relevant to your task.